### ISSUE TITLE
JAWS + Chrome ignores figures if image has empty alt / lacks alt

### ISSUE THREAD

**Issue by scottaohara** [FreedomScientific/standards-support#161]
### Summary
If a `figure` contains an image with an empty `alt` or lack of `alt`, JAWS + Chrome will ignore the entire `figure`, even if it has a `figcaption`.

With Chrome + JAWS 2019 w/default settings:
* Go to [Test 2 of these figure/figcaption examples](https://scottaohara.github.io/testing/figure/tests.html).
* Use the virtual cursor to attempt to navigate the contents of the figure to announce the figcaption's content.

### Expected result
The figure grouping would have been announced with the figcaption as the accessible name.  The image would have been announced as earth.jpg, or ignored, but the caption would have been discoverable.

### Actual result
JAWS + Chrome completely ignore the figure and its caption.  Virtual cursor cannot focus/announce the figcaption's content.

### Example
[Test 2 of these figure/figcaption examples](https://scottaohara.github.io/testing/figure/) demonstrates this behavior with JAWS default settings.

### Additional Information
@stevefaulkner found the following from JAWS inspect w/Chrome:
```
<domnode inType="Figure" wiid="P125" fsType="120" fsTag="FIGURE" fsID="ffff9a18" fsExclude="1" fsStructural="1" fsOffset="879" fsEnd="904" fsLeft="72" fsRight="1823" fsTop="490" fsBottom="590" fsPrompt="Content caption." role="figure">
```

#### JAWS version and build number
JAWS 2019 latest release

 
#### Operating System and version
Windows 10 Pro
10.0.17134

#### Browser and version:
Chrome 71

**Comment by jducrot** [comment]
@scottaohara I ran through the first couple of tests using Chrome 83 + (JAWS 20 or NVDA  2020.1) and there are some improvements. I'd be happy to contribute data if you could expose the code. Maybe a TPG pen? Let me know

**Comment by scottaohara** [comment]
@jducrot made the test file public here: 

https://scottaohara.github.io/tests/html-figure-figcaption/screen-reader-results/

https://github.com/scottaohara/tests/tree/master/html-figure-figcaption/screen-reader-results

if you would like to add some updates, a PR would be more than welcome

**Comment by scottaohara** [comment]
testing again today, latest chrome and jaws still ignore the mentioned test case.

**Comment by scottaohara** [comment]
from talking with @stevefaulkner today, @kurtjwolf and @BenKeyFSI could you please look into this.

**Comment by mgifford** [comment]
Would love to get an update on this. Seems that NVDA just fixed the issues that they had with figcaption recently.

**Comment by aardrian** [comment]
Is this issue resolved? Rather, is the issue where it ignores the entire thing resolved?

Using JAWS 2021.2107.2 with Chrome 97.0.4692.99, if I go to [Test 2](https://scottaohara.github.io/testing/figure/tests.html#:~:text=Pattern%20notes%3A-,Test%202,-Content%20caption.) that Scott linked, I get the following:

Pressing down arrow from the heading:
> group start Content caption.

Then I press down arrow to the image:
> To get missing image descriptions, open the context menu. Unlabeled graphic

Then I press down arrow again, moving me to the caption:
> Content caption.

Then I press down arrow again and hear:
> group end

I made my own test and it also behaves as expected, announcing the `alt`-less figure just fine (with the added verbosity of the group announcement).

**Comment by selfthinker** [comment]
I have also just retested this today and found that it is fixed.
I used [this test page](https://output.jsbin.com/vuyirud) with JAWS 2022.2207.25 in Chrome 103.0.560.134 on Windows 10.

Although the output is different, which I guess might be due to JAWS 2022: "entering details [caption] has details" where "[caption]" is the content of the caption.

**Comment by selfthinker** [comment]
Actually, I've tried to test this in older versions of JAWS and it was working as far back as JAWS 2019. (I could go further back but stopped because I knew for sure JAWS 2019 wasn't working.) I wonder if it isn't JAWS that has fixed something but Chrome?

**Comment by stevefaulkner** [comment]
tested with JAWS 2023.2307.37, APPEARS TO WORK AS EXPECTED. closing

**Comment by samthill** [comment]
This appears to be back

JAWS 2025.2410.106
Chrome 131.0.6778.86 (Official Build) (64-bit)
Windows 11 Pro Version	10.0.22631 Build 22631
